internal/service/fis/experiment_template.go

```diff
@@ -801,15 +801,18 @@ func validExperimentTemplateStopConditionSource() schema.SchemaValidateFunc {
 }
 
 func validExperimentTemplateActionTargetKey() schema.SchemaValidateFunc {
+	// See https://docs.aws.amazon.com/fis/latest/userguide/actions.html#action-targets
 	allowedStopConditionSources := []string{
 		"Cluster",
```

Comment:
This does not look like a correct target, but removing this would probably be a breaking change.